lib/main_common.pm

```diff
@@ -1083,36 +1083,39 @@ sub load_consoletests {
-        # TODO test on openSUSE https://progress.opensuse.org/issues/31972
-        if (is_sle && sle_version_at_least('12-SP1')) {    # shibboleth-sp not available on SLES 12 GA
-            loadtest "console/shibboleth";
-        }
-        if (!is_staging && (is_opensuse || get_var('ADDONS', '') =~ /wsm/ || get_var('SCC_ADDONS', '') =~ /wsm/)) {
+        # disable these tests of server packages for SLED (poo#36436)
+        if (!is_desktop()) {
+            if (check_var('BACKEND', 'qemu') && !is_jeos) {
+                # The NFS test expects the IP to be 10.0.2.15
+                loadtest "console/yast2_nfs_server";
+            }
+            loadtest "console/http_srv";
+            loadtest "console/dns_srv";
+            loadtest "console/postgresql_server" unless (is_leap('<15.0'));
             # TODO test on openSUSE https://progress.opensuse.org/issues/31972
-            loadtest "console/pcre" if is_sle;
-            # TODO test on SLE https://progress.opensuse.org/issues/31972
-            loadtest "console/mysql_odbc" if is_opensuse;
-            if (is_leap('<15.0') || is_sle('<15')) {
-                loadtest "console/php5";
-                loadtest "console/php5_mysql";
-                loadtest "console/php5_postgresql96";
+            if (is_sle && sle_version_at_least('12-SP1')) {    # shibboleth-sp not available on SLES 12 GA
+                loadtest "console/shibboleth";
+            }
+            if (!is_staging && (is_opensuse || get_var('ADDONS', '') =~ /wsm/ || get_var('SCC_ADDONS', '') =~ /wsm/)) {
+                # TODO test on openSUSE https://progress.opensuse.org/issues/31972
+                loadtest "console/pcre" if is_sle;
+                # TODO test on SLE https://progress.opensuse.org/issues/31972
+                loadtest "console/mysql_odbc" if is_opensuse;
+                if (is_leap('<15.0') || is_sle('<15')) {
+                    loadtest "console/php5";
+                    loadtest "console/php5_mysql";
+                    loadtest "console/php5_postgresql96";
+                }
+                loadtest "console/php7";
+                loadtest "console/php7_mysql";
+                loadtest "console/php7_postgresql96";
             }
-            loadtest "console/php7";
-            loadtest "console/php7_mysql";
-            loadtest "console/php7_postgresql96";
+            # TODO test on openSUSE https://progress.opensuse.org/issues/31972
+            loadtest "console/apache_ssl" if is_sle;
+            # TODO test on openSUSE https://progress.opensuse.org/issues/31972
+            loadtest "console/apache_nss" if is_sle;
         }
-        # TODO test on openSUSE https://progress.opensuse.org/issues/31972
-        loadtest "console/apache_ssl" if is_sle;
-        # TODO test on openSUSE https://progress.opensuse.org/issues/31972
-        loadtest "console/apache_nss" if is_sle;
+        loadtest "console/mysql_srv";
```

Comment:
I'm still not sure, is it really just a single package we want to test on SLED? @okurz WDYT?

Reply:
Yes, if there is just mysql_srv working on SLED then it's just it.